docs/cloud/index.md

```diff
@@ -4,11 +4,11 @@ title: Temporal Cloud documentation
 sidebar_label: Temporal Cloud
 ---
 
-Temporal Cloud is a fully managed public cloud service for the [Temporal Platform](/concepts/what-is-the-temporal-platform/), offered as an alternative to deploying and operating your own [Temporal Cluster](/concepts/what-is-a-temporal-cluster/).
+Temporal Cloud is a fully managed public cloud service for the [Temporal Platform](/concepts/what-is-the-temporal-platform), offered as an alternative to deploying and operating your own [Temporal Cluster](/concepts/what-is-a-temporal-cluster).
 
 :::tip Updated UI
 
-Temporal Cloud is now using Temporal UI v2 as the default UI.
+Temporal Cloud is now using [Temporal Web UI v2](/web-ui) as the default UI.
```

Comment:
This will exist as soon as this lands: https://github.com/temporalio/documentation/pull/1257